modal/cli/environment.py

```diff
@@ -1,5 +1,6 @@
 # Copyright Modal Labs 2023
-from typing import Optional
+from os import environ
```

Comment:
Extremely nitpicky but directly importing `os` and then doing `os.environ` would be more conventional.